code/game/objects/items.dm

```diff
@@ -1017,6 +1017,11 @@ var/global/list/image/blood_overlays = list()
 	if(flags & INVULNERABLE)
 		return
 	spawn(0) //this is needed or multiple items will be thrown sequentially and not simultaneously
+		if(anchored)
```

Comment:
I found this when testing the gravity effect, very small singularities would move anchored items like intercoms, this made it more consistent with general objects